protocol.html

```diff
@@ -717,6 +719,62 @@ <h4 property="schema:name"><span>Note</span>: Conditional Update</h4>
+
+                    <p><span about="" id="server-patch-n3-semantics" rel="spec:requirement" resource="#server-patch-n3-semantics"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> process a patch resource against the target document as follows:</span></span></p>
+                    <ol>
+                      <li>Start from the RDF dataset in the target document, or an empty RDF dataset if the target resources does not exist yet.</li>
+                      <li>If <code>?conditions</code> is non-empty, find all (possibly empty) variable mappings such that all of the resulting triples occur in the dataset. If no such mapping exists, or if multiple mappings exist, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.
+                      <li>The resulting variable mapping is propagated to the <code>?deletions</code> and <code>?insertions</code> formulae to obtain two sets of resulting triples.</li>
+                      <li>If the set of triples resulting from <code>?deletions</code> is non-empty and the dataset does not contain <em>all</em> of these triples, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.
+                      <li>The triples resulting from <code>?deletions</code> are to be removed from the RDF dataset.</li>
+                      <li>The triples resulting from <code>?insertions</code> are to be added to the RDF dataset, with each blank node from <code>?insertions</code> resulting in a newly created blank node.</li>
+                      <li>The combination of deletions followed by insertions then forms the new resource state of the RDF document, and the server responds with the appropriate status code.</li>
+                    </ol>
+                  </div>
+
+                  <figure id="n3-patch-example" class="example listing" rel="schema:hasPart" resource="#n3-patch-example">
+                    <p class="example-h"><span>Example</span>: Applying an N3 patch.</p>
+                    <pre about="#n3-patch-example" property="schema:description" typeof="fabio:Script">
+<code>@prefix solid: &lt;http://www.w3.org/ns/solid/terms#&gt;.
+@prefix ex: &lt;http://www.example.org/terms#&gt;.
+
+_:rename a solid:Patch;
+   solid:where   { ?person ex:familyName "Anchor". };
+   solid:inserts { ?person ex:givenName "An". };
+   solid:deletes { ?person ex:givenName "Bob". }.</code></pre>
+                    <figcaption property="schema:name">This N3 Patch instructs to rename <em>An Anchor</em> into <em>Bob Anchor</em>, on the condition that no other Anchor family members are present in the target RDF document.</figcaption>
```

Comment:
I admit that it took me a couple of eyeballing to get "An". Can we use Kathryn Janeway instead or whatever you prefer? (Related: https://github.com/solid/specification/issues/244 )

If I'm not mistaken, the intention is to rename "Bob Anchor" to "An Anchor":

```suggestion
                    <figcaption property="schema:name">This N3 Patch instructs to rename <em>Bob Anchor</em> into <em>An Anchor</em>, on the condition that no other Anchor family members are present in the target RDF document.</figcaption>
```

Reply:
Ah.  Might I request the more common name spellings of "Ann" or "Anne" for this use?

Reply:
Done in fef2ac79e99f497c769df416aa7be968e254c349 (meant to use traditional Alice/Bob example).